cmd/bnsd/app/app_nft_test.go

```diff
@@ -0,0 +1,66 @@
+package app_test
+
+import (
+	"testing"
+
+	"github.com/iov-one/weave/cmd/bnsd/app"
+	"github.com/iov-one/weave/cmd/bnsd/app/testdata/fixtures"
+	"github.com/iov-one/weave/x/nft/blockchain"
+	"github.com/iov-one/weave/x/nft/ticker"
+	"github.com/iov-one/weave/x/nft/username"
+	"github.com/stretchr/testify/require"
+)
+
+func TestIssueNfts(t *testing.T) {
+	appFixture := fixtures.NewApp()
+	isuserAddr := appFixture.GenesisKeyAddress
+	issuerPrivKey := appFixture.GenesisKey
+
+	myApp := appFixture.Build()
+	myBlockchainID := []byte("myblockchain")
+
+	// when blockchain nft issued
+	tx := &app.Tx{
+		Sum: &app.Tx_IssueBlockchainNftMsg{&blockchain.IssueTokenMsg{
+			Id:      myBlockchainID,
+			Owner:   isuserAddr,
+			Details: blockchain.TokenDetails{},
+		},
+		},
+	}
+
+	res := signAndCommit(t, myApp, tx, []Signer{{issuerPrivKey, 0}}, appFixture.ChainID, 2)
+
+	// then
+	require.EqualValues(t, 0, res.Code)
+
+	// and when username nft issued
+	tx = &app.Tx{
+		Sum: &app.Tx_IssueUsernameNftMsg{&username.IssueTokenMsg{
+			Id:      []byte("anybody@example.com"),
+			Owner:   isuserAddr,
+			Details: username.TokenDetails{[]username.ChainAddress{}},
```

Comment:
it would be cool to refer to the exisiting blockchain here, like you do with the ticker.

but great to have the basic cases covered to check the full stack